wiki/Game_modifier/Auto/fr.md

```diff
@@ -0,0 +1,61 @@
+---
+stub: true
+tags:
+  - auto
+  - autoplay
+  - auto play
+  - AT
+  - AO
+  - mod
+  - game modifier
+---
+
+# Le mod Auto
+
+![Icône du mod AT](/wiki/shared/mods/AT.png "Icône du mod Auto (AT)")
+
+*Pour la liste complète de tous les mods, voir : [Modificateur de jeu](/wiki/Game_modifier)*\
+*Ne pas confondre avec le mod [Autopilot](/wiki/Game_modifier/Autopilot)*.
+
+## À propos
+
+- Abréviation : AT ou AO
+- Type : Spécial
+- Multiplicateur de score : 1.00x
```

Comment:
Isn't it 0.00x ?

Reply:
The value is original on the English version, I don't know if it can be wrong.

Reply:
yeah that